trymito.io/styles/[slug].module.css

```diff
@@ -1,19 +1,71 @@
+.decorative_line {
+    position: absolute;
+    width: 1px;
+    height: calc(100% - 42rem);
+    border: 0.5px dashed rgba(208, 208, 208, 0.25);
+    left: max(19.5rem, calc(((100vw - 70rem) / 2) - 6px));
+}
 
+/* Post Styling */
 .post {
-    margin-top: 4rem;
+    margin-top: 7rem;
     margin-bottom: 4rem;
+    width: 100%;
+    font-family: Inter, Ubuntu, Helvetica, Arial, sans-serif;
+}
+
+.post a, .suggested_post h4 {
+    text-decoration: underline 0.15em rgba(0, 0, 0, 0);
+    transition: text-decoration-color 300ms;
+}
+  
+.post a:hover, .suggested_post:hover h4 {
+    text-decoration-color: rgba(0, 0, 0, 1);
+}
+
+
+
+/* Post Banner */
+.post_banner {
+    background-color: var(--color-purple);
+    padding: 5rem 20rem;
+    width: 100%;
+    display: flex;
+    justify-content: center;
 }
 
 .post_title {
-    max-width: min(800px, 90vw);
+    display: flex;
+    justify-content: flex-start;
+    flex-direction: column;
 }
 
 .post_title h1 {
     font-size: 3rem;
+    max-width: 70rem;
 }
 
+.post_title p, .post_title h1 {
+    margin-top: 0
+}
+
+.post_metadata {
+    display: flex;
+    flex-direction: row;
+    margin-bottom: .5rem;
+}
+
+.post_additional_info {
+    margin-left: 3em;
+}
+
+
+
+/* Post Content */
 .post_content {
-    max-width: min(720px, 90vw);
+    margin: 0 2rem;
+    max-width: 70rem;
+    width: calc(100vw - 40rem);
```

Comment:
I removed the width calc and it seems to be working exactly the same. Lmk your thoughts

Reply:
Ah yeah it's actually not working the same - I added this because for some reason this happens when the content should be less than 70rem:
![image](https://github.com/mito-ds/mito/assets/6673460/fd48901c-bd04-4a7b-b044-2529b5fb6b14)

Reply:
I really wanted the blog to be responsive to the page width and for some reason flexbox was just not doing it as expected here. I'm not sure what was throwing it off but setting the explicit width fixed it. Let me know if you know of any potential solutions to this using the flexbox style options

Reply:
Makes sense! I appreciate your care for the polish here!